eng/testing/tests.mobile.targets

```diff
@@ -132,6 +132,9 @@
       <WasmFilesToIncludeInFileSystem Include="@(ContentWithTargetPath)" />
       <WasmFilesToIncludeInFileSystem Include="@(ReferenceCopyLocalPaths)" Condition="'%(ReferenceCopyLocalPaths.BuildReference)' == 'true' and !$([System.String]::new('%(ReferenceCopyLocalPaths.Identity)').EndsWith('.resources.dll'))" />
       <WasmFilesToIncludeInFileSystem Include="@(WasmSatelliteAssemblies)" TargetPath="%(WasmSatelliteAssemblies.CultureName)\%(WasmSatelliteAssemblies.Filename)%(WasmSatelliteAssemblies.Extension)" />
+      <WasmFilesToIncludeInFileSystem Include="$(PublishDir)System.Private.CoreLib.dll" Condition="'$(AssemblyName)' == 'System.Reflection.MetadataLoadContext.Tests'" />
```

Comment:
I thought we already mapped all of the framework assemblies to the VFS

Reply:
no, that was undone as part of the ICU PR which included some asset loading changes.

Reply:
Ah I see. I thought we were mapping the whole thing. What is going to be the story for customers if they try to load an assembly? i.e: `Load(typeof(string).Assembly.Location)`?

Reply:
right now it will fail. we've talked about intercepting any file open of a BCL assembly path and returning the contents of the in memory byte array.

Reply:
Ok... sounds good.